docs/documentation/platform/identities/universal-auth.mdx

```diff
@@ -154,23 +157,45 @@ using the Universal Auth authentication method.
   </Step>
 </Steps>
 
-**FAQ**
+## Solving Secret Zero with Periodic Tokens
 
-<AccordionGroup>
-<Accordion title="Why is the Infisical API rejecting my identity credentials?">
-  There are a few reasons for why this might happen:
+A common challenge in cloud-native and automated environments is the "secret zero" problem: how to securely bootstrap a workload (such as a VM, container, or serverless function) with its first credential, without hard-coding static secrets or requiring manual intervention.
+
+**Periodic tokens** in Universal Auth solve this by allowing you to issue a renewable, non-expiring access token to your workload:
+
+- When you set the **Access Token Period** in the Universal Auth configuration, the issued access token becomes renewable for the specified period (in seconds).
+- The token can be renewed any number of times, each time for the same period, with no maximum lifetime (unless you set a use limit).
+- As long as the token is renewed before its period expires, it remains valid and does not require re-issuing static credentials.
+- TTL and Max TTL are ignored when Access Token Period is set.
 
-  - The client secret or access token has expired.
-  - The identity is insufficiently permissioned to interact with the resources you wish to access.
-  - The client secret/access token is being used from an untrusted IP.
-</Accordion>
-<Accordion title="What is access token renewal and TTL/Max TTL?">
-  A identity access token can have a time-to-live (TTL) or incremental lifetime after which it expires.
+### Example: Bootstrapping with a Periodic Token
```

Comment:
i think we missed the important part where they set the client secret access to something low like `1` for num uses or low ttl so that once the machine identity i logged it, any distrubance between its period would cause it to stop functioning